tsconfig.json

```diff
@@ -28,6 +28,8 @@
       "@mui/x-tree-view-pro/*": ["./packages/x-tree-view-pro/src/*"],
       "@mui/x-license": ["./packages/x-license/src"],
       "@mui/x-license/*": ["./packages/x-license/src/*"],
+      "@mui/docs": ["./node_modules/@mui/monorepo/packages/mui-docs/src"],
+      "@mui/docs/*": ["./node_modules/@mui/monorepo/packages/mui-docs/src/*"],
```

Comment:
@LukasTy Maybe this could impact the pnpm

Reply:
I'll double-check after updating on pnpm side. 👌